dining/views.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime
+
+from django.http import HttpResponseForbidden, JsonResponse
+from legacy.models import Account, DiningBalance, DiningTransaction
+from options.models import get_option, get_value
+from rest_framework.views import APIView
+
+from studentlife.utils import date_iso_eastern, get_new_start_end
+
+
+# Create your views here.
+
+
+class Dashboard(APIView):
+    def get(self, request, format=None):
+        if not self.request.user.is_authenticated:
+            return HttpResponseForbidden()
+
+        self.pennid = "76627463"  # request.user.username
+        self.uid = Account.objects.filter(pennid=self.pennid)[0].id
```

Comment:
```suggestion
        try:
             self.uid = Account.objects.get(pennid=self.pennid).id
        catch Account.DoesNotExist:
            pass # some code here in case the pennID doesn't exist
```

Reply:
also realized the indentation on that suggestion may be messed up :/ but generally if you're looking for one element it's better to use `get()` than `filter()[0]`.